lms/djangoapps/certificates/migrations/0025_cleanup_certificate_errors.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.20 on 2021-04-27 18:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    """
+    Clean records in the `CERTIFICATES_GENERATEDCERTIFICATE` table that are in the downloadable state but also have
+    old errors still part of their certificate record.
+
+    As part of this migration we are also altering the Managers for the GeneratedCertificate model. We need the ability
+    to access the `objects` attribute for the cleanup.
+    """
+    def cleanup_certificate_records(apps, schema_editor):
+        GeneratedCertificate = apps.get_model('certificates', 'GeneratedCertificate')
+
+        cert_list = (
+            GeneratedCertificate.objects
+            .filter(status='downloadable')
+            .exclude(error_reason='')
+        )
+
+        for cert in cert_list:
+            cert.error_reason = ''
+            cert.save()
+
+    dependencies = [
+        ('certificates', '0024_delete_allowlistgenerationconfiguration'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='generatedcertificate',
+            managers=[
+            ],
+        ),
+        migrations.RunPython(cleanup_certificate_records, reverse_code=migrations.RunPython.noop)
```

Comment:
I originally intended to have two migrations -- first to update the object Managers with the second to clean up the data. Django didn't like this. I ran into the following error:

CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0026_cleanup_generatedcertificate_errors, 0025_update_generatedcertificate_managers in certificates).
To fix them run 'python manage.py makemigrations --merge'
Running the suggested command created a third migration... so I decided to try and just combine the two migrations into one, which worked fine locally.